.chloggen/add-shellcheck-github-actions-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```

Comment:
We don't need a changelog for this since it only affects our CI and not the end-users. I will add the skip changelog to make the changelog check pass, could you remove this file?